src/pages/AppDetail.svelte

```diff
@@ -8,10 +8,13 @@
   import Pill from "../components/Pill.svelte";
   import { TabGroup, Tab, TabContent } from "../components/tabs";
   import PageTitle from "../components/PageTitle.svelte";
+  import { pageTitle } from "../state/stores";
 
   export let params;
 
   const appDataPromise = getAppData(params.app);
+
+  pageTitle.set(`${params.app} | Apps `);
```

Comment:
@wlach Should I change this to just ` pageTitle.set(params.app);`

Reply:
Yes please! Sorry, missed that one.